config/routes.rb

```diff
@@ -38,7 +38,7 @@
     get "/debug" => "debug#index", as: :debug_document
 
     get "/search-contacts" => "contacts#search", as: :search_contacts
-    post "/search-contacts" => "contacts#insert", as: :insert_contact
+    post "/search-contacts" => "contacts#search", as: :insert_contact
```

Comment:
Can we rename the 'as' part here? I think we can actually leave it out for both of them and use 'search_contacts_path'.

Reply:
Updated and removed both